### ISSUE TITLE
Type 'EChartsOption' is not assignable to type 'xxxxxx'

### ISSUE THREAD

**Issue by YamadaAoi** [fi3ework/vite-plugin-checker#141]
### Describe the bug

when I use  vue3 reactive define a const paramter
like this: 

`
import { EChartsOption } from 'echarts'

interface ChartsOption {
  chartId: string
  option?: EChartsOption
}

function getOption(): EChartsOption {
  return {
    ... // some echarts options
  }
}

const chartOption = reactive<EChartsOption>({ chartId: 'xxx' })

chartOption.option = getOption() // get unexpected type error

`
![image](https://user-images.githubusercontent.com/37076519/174035071-de3f30f9-85b5-459f-86a1-730cd46afc9d.png)

type should be same but ...

### Reproduction

try it

### Expected behavior

type should be same

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
    Memory: 11.26 GB / 31.79 GB
  Binaries:
    Node: 16.14.0 - D:\Program Files\nodejs\node.EXE
    Yarn: 1.22.18 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.3.1 - D:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (102.0.1245.39)
    Internet Explorer: 11.0.22000.120
  npmPackages:
    vite-plugin-checker: ^0.4.6 => 0.4.6
```

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
